charts/application/templates/k8s-rolebinding.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if or (eq .Values.k8sAPI.role "edit") (eq .Values.k8sAPI.role "view") (eq .Values.k8sAPI.role "admin") }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ $.Release.Name }}
```

Comment:
I think all other labels https://github.com/MediaMarktSaturn/helm-charts/blob/main/charts/application/values.yaml#L93-L94 should be added to this resource as well.